### PR TITLE
Linter fixes to enable linter commit checks

### PR DESCRIPTION
**Pull request by rdlaitila**
This PR includes fixes for linting errors found during development of IaC CI/CD pipeline commit checks

### CODE REVIEW

File: files/tools/iac_tooling_version.sh [hunk 1]
```diff
@@ -9,16 +9,16 @@ function error_exit() {
...
 TERRAFORM_INFO="$(terraform version -json)"
-TERRAFORM_VERSION=$(echo $TERRAFORM_INFO | jq .terraform_version )
-TERRAFORM_REVISION=$(echo $TERRAFORM_INFO | jq .terraform_revision )
-PROVIDER_SELECTIONS=$(echo $TERRAFORM_INFO | jq -c .provider_selections )
-TERRAFORM_OUTDATED=$(echo $TERRAFORM_INFO | jq .terraform_outdated )
+TERRAFORM_VERSION=$(echo "$TERRAFORM_INFO" | jq .terraform_version )
+TERRAFORM_REVISION=$(echo "$TERRAFORM_INFO" | jq .terraform_revision )
+PROVIDER_SELECTIONS=$(echo "$TERRAFORM_INFO" | jq -c .provider_selections )
+TERRAFORM_OUTDATED=$(echo "$TERRAFORM_INFO" | jq .terraform_outdated )
 # echo "${TERRAFORM_INFO | jq "
```
Comment (thpang) [inline files/tools/iac_tooling_version.sh:22]: Can this line be removed? It's incorrect in its current form. No ending "}"

Comment (rdlaitila) [reply files/tools/iac_tooling_version.sh]: line removed

**Comment by thpang** [comment]
@rdlaitila did you make progress on the lint updates from our meeting yesterday?

**Comment by rdlaitila** [comment]
@thpang  I'm working on linter configuration files today and pushed updates for hadolint. Additional configuration files for tflint and shellcheck should be done shortly.

**Comment by rdlaitila** [comment]
@iac-bot deploy

**Comment by rdlaitila** [comment]
@iac-bot deploy

**Comment by rdlaitila** [comment]
@iac-bot deploy

**Comment by rdlaitila** [comment]
@iac-bot deploy

**Comment by rdlaitila** [comment]
@thpang  I've enabled the linter checks repo-wide. This PR is ready for review and merge.

**Review by thpang — APPROVED** [review]
Looks good to me. Go for it.

**Pull request merged**